iree/tools/vulkan_gui_util.h

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Google LLC
```

Comment:
doesn't really seem right for this to be under tools/ (directory is getting crowded) - what is the intent of this? can it be moved under iree/testing/vulkan/ to denote that it's purely test related?

Reply:
`iree/testing/vulkan` would also work. The intent was to share between the Vulkan GUI sample and a new GUI program like `iree-run-module` for running models through vendor-specific tools like NVIDIA Nsight.

Reply:
Yeah `iree/testing/vulkan` works for me. Another option might be `iree/tools/utils`.